RecoLocalFastTime/FTLCommonAlgos/plugins/MTDRecHitAlgo.cc

```diff
@@ -22,14 +26,20 @@ class MTDRecHitAlgo : public MTDRecHitAlgoBase {
   FTLRecHit makeRecHit(const FTLUncalibratedRecHit& uRecHit, uint32_t& flags) const final;
 
 private:
-  double thresholdToKeep_, calibration_;
+  std::vector<double> thresholdToKeep_;
+  double calibration_;
   const MTDTimeCalib* time_calib_;
+  const MTDTopology* topology_;
+  static constexpr int topologycode1Disk_ = 4;
```

Comment:
@gsorrentino18 @casarsa there is no need to define hardcoded thresholds, you may simply use the MTDTopologyMode enum class with something like

```
if (mtdTopologyMode <= static_cast<int>(MTDTopologyMode::Mode::barphiflat)) {
```
as we are doing elsewhere in the code (and I am using in the RecoMTD/DetLayers code I am preparing)